qa-heal.planx-pla.net/portal/gitops.json

```diff
@@ -189,6 +208,7 @@
     "tagSelector": {
       "showTagCategoryNames": true
     },
+    "studies": [],
```

Comment:
what is this?

Reply:
This is for adding links to tutorial notebooks. Its implemented in BRH as a list of dicts, see https://github.com/uc-cdis/gitops-qa/pull/2193/files. In HEAL,  since we are not doing that right now this is initialized as an empty list. @johnfrancismccann can add more info to this

Reply:
🤦 these new fields that we have added into portal config should be reflected in https://github.com/uc-cdis/data-portal/blob/master/docs/portal_config.md so ppl can know what they are for and how to use them

Reply:
Ok, I wasn't aware of that

Reply:
I'll find some time today to make that PR and tag you